### DISCUSSION TITLE
How do you add badges for the run?

### DISCUSSION THREAD

**Discussion by zaad97** [jynnie/soullocke#9] [Q&A] [answered]
Hello. First of all, I would like to say thank you for making this awesome website.

For my question, I see there is badges. However, i don't know how to add them in the run.
Is it possible to add right now (if so, how?) or is the code not implemented yet?

**Comment by jynnie** [comment, marked as answer]
Hi! Thank you! I'm glad it's useful! You can add Badges with the same plus you use to add locations. I'll see if I can make a clearer way to add them.

**Reply by zaad97** [reply]
Oh, i found them. Maybe if you can toggle them in  the Summary Tab, it will also add/delete in the Timeline Tab.

**Reply by jynnie** [reply]
Yeah, that's a great idea! I'll look into doing that soon
